src/gpu/intel/jit/gemm/gen_gemm_kernel.cpp

```diff
@@ -865,7 +865,7 @@ void gen_gemm_kernel_t::init_interface() {
     if (problem.cOffset != COffset::None || problem.sumA || problem.sumB) {
         interface_.newArgument(
                 "CO", ExternalArgumentType::GlobalPtr, co_access);
-        interface_.newArgument("offset_CO", DataType::d);
+        interface_.newArgument("offset_CO", DataType::q);
```

Comment:
Does the kernel need to be altered on the generator side to interpret this argument (and the others below) as qw arguments?

Reply:
No, the declared data type for the argument (q) will be used in the kernel automatically.